icons/test-tube.json

```
@@ -0,0 +1,10 @@
+{
+  "$schema": "../icon.schema.json",
+  "tags": [
+    "tube",
```

Nice one! We could maybe add some (semi)synonyms for easier finding:

```suggestion
    "tube",
    "vial",
    "phial",
    "flask",
    "ampoule",
    "ampule",
```